### ISSUE TITLE
Seeing effect of GAN training

### ISSUE THREAD

**Issue by MartinLNG** [MartinLNG/adversarial_training_mps#99]
To see an effect of GAN training design an experiment that risks destroying the classifier by
- ignore retraining
- large learning rate
- GAN loss functions
